flow/pom.xml

```diff
@@ -56,6 +56,31 @@
                     <finalName>aws-flow-java</finalName>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
why is this plugin required in modules like flow, services. Isn't just http-clients and core enough?

Reply:
I'm not sure what you mean. All of the dependencies pulled in by a service client need to be OSGi compatible, which includes flow.